core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -344,6 +342,8 @@
   <script src="/templates/dev/head/services/SpeechSynthesisChunkerService.js"></script>
   <script src="/templates/dev/head/services/StateTopAnswersStatsBackendApiService.js"></script>
   <script src="/templates/dev/head/services/StateTopAnswersStatsService.js"></script>
+  <script src="/templates/dev/head/services/ExplorationFeaturesService.js"></script>
+  <script src="/templates/dev/head/services/ExplorationFeaturesBackendApiService.js"></script>
```

Comment:
As we are generalizing this in the backend, would it make sense to make this FeaturesBackendApiService? Or shall we keep this service specific to exploration features (as it gets data from an exploration specific URL). 

Just wanted your opinion here. Happy to leave it as is too.